.github/workflows/main.yml

```diff
@@ -52,10 +52,10 @@ jobs:
       run: brew install gcc@${GCC_V} || brew upgrade gcc@${GCC_V} || true
 
     - name: Configure with CMake
-      run: cmake -Wdev -S . -B . # We should build out of source but loadtxt needs fixing
+      run: cmake -Wdev -S . -B build # We should build out of source but loadtxt needs fixing
```

Comment:
```suggestion
      run: cmake -Wdev -S . -B build
```

Reply:
Actually, this comment should be deleted if the issue is fixed and being tested.